NuGet/OpenRiaServices.Hosting.WCF/tools/Install.ps1

```diff
@@ -15,7 +15,7 @@ try {
 
     function RemoveNodesWithOldNamespaces($nodes) {
         foreach($node in $nodes)  {
-            if ($node.Attributes["type"].Value.Contains("DomainServices")) {
+            if ($node.Attributes["type"].Value.Contains(".DomainServices.")) {
```

Comment:
Was there something in the `OpenRiaServices.DomainServices` namespace ? This would not catch it.

Reply:
I do not think there was anything directly in the namespace, but it would catch it since the type name would follow after the final dot